maquetado/scss/details.scss

```diff
@@ -1,14 +1,37 @@
 .book,
 .book-element,
-.book-elements {
+.book-elements,
+.book-title {
   display: flex;
 }
 
+.book {
+  margin: 10px;
+  box-shadow: 2px 5px 10px;
+}
+
+.book-element {
+  margin-bottom: 30px;
+}
+
+.book-element-heading {
+  margin-right: 4px;
+}
+
 .book-info {
   flex-direction: column;
+  width: 25%;
 }
 
 .book-image {
   height: 562px;
   width: 398px;
+  margin: 10px;
+}
+
+.book-title {
+  margin-top: 10px;
```

Comment:
Avoid using margin top. You can use padding top in your container instead `book-info`